relion/protocols/protocol_extract_particles.py

```diff
@@ -264,11 +256,11 @@ def _getExtractArgs(self):
         params += ' --extract_size %d' % self.boxSize
 
         if self.backDiameter <= 0:
-            diameter = self.boxSize.get() * 0.75 / self._getDownFactor()
+            diameter = self.boxSize.get() * 0.75
         else:
             diameter = self.backDiameter.get()
 
-        params += ' --bg_radius %d' % int(diameter/2)
+        params += ' --bg_radius %d' % int(diameter/(2 * self._getDownFactor()))
```

Comment:
this means user interaction has changed! Users have to provide diameter before rescaling! It was not the case in 1.2.1! Maybe we should put this into label text!

Reply:
Do we really need to put this? We are not longer supporting Relion 1.x anyway!

Reply:
I meant scipion 1.2.1

Reply:
Ah, Ok...still I think it was a bug (or inconsistency) in our side in respect with Relion GUI...but I agree that maybe a label could help.